manifests/klusterlet/managed/klusterlet-registration-clusterrole-addon-management.yaml

```diff
@@ -7,6 +7,13 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: open-cluster-management:{{ .KlusterletName }}-registration:addon-management
+  labels:
+    createdBy: klusterlet
```

Comment:
considering there might be multiple klusterlet in hosted mode, should we put the klusterlet as the real klusterlet name?

Reply:
the label `createdBy: klusterlet` is used to filter in deployment informer, I added another label klusterletName to set the real klusterlet name.

Reply:
TBH maybe not necessary to create both label right now...why do we still need createdBy: klusterlet if we have klusterletName label? We can change the filter in deployment informer.

Reply:
how about to change the label to `createdByKlusterlet: <klusterlet name>` , in deployment informer, changed to check if the key `createdByKlusterlet` exists?